lib/bindings/HTTPBinding.js

```diff
@@ -53,6 +54,19 @@ const { promisify } = require('util');
 const json = promisify(bodyParser.json({ strict: false })); // accept anything JSON.parse accepts.
 const text = promisify(bodyParser.text());
 const raw = promisify(bodyParser.raw());
+const xml2js = require('xml2js');
+const xmlStripPrefix = xml2js.processors.stripPrefix;
+const xml = promisify(bodyParser.xml({
+        xmlParseOptions: {
+            // XML namespaces might change from one request to the next.
+            // It is useful to remove them from the document,
+            // to be able to refer to tags later in JEXL transformations.
+            // See https://github.com/Leonidas-from-XIV/node-xml2js/issues/87
```

Comment:
Maybe this should be explained in documentation?

Reply:
Done in https://github.com/telefonicaid/iotagent-json/pull/771/commits/402e20064ee5bb73c62015b95ec47108f6a97570